CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# How to contribute to Concuerror
+
+First of all, thank you for taking time to contribute to Concuerror!
+
+
+## I just have a question!
+
+Use the [mailing list][mailing list] and take a look at the [FAQ][faq] if you haven't already!
+
+
+## Did you encounter a bug?
+
+* **Ensure the bug has not already been reported** by searching Github's [Issues][Issues]. If you find a relevant issue, feel free to leave a new comment there.
+
+* If you cannot find an existing relevant Issue, [open a new Issue][open a new Issue]. Include a **title** and a **description**, as much relevant information as possible, and a **code sample** or an **executable test case** demonstrating the expected behavior that is not occurring.
+
+
+## Did you write a patch that fixes a bug?
+
+* Fork Concuerror's repository and push a new branch for your patch, using a good, descriptive name.
+
+* Open a Pull Request (PR) with the patch.
+
+* Ensure the PR's description explains the problem and solution. Include the relevant issue number, if applicable.
+
+
+## Do you want to add a new feature or change an existing one?
+
+* Take a look at the Issue pages of the developers' own forks for relevant ideas and leave comments there: [@aronisstav Issues][aronisstav Issues]
+
+* Suggest your change in the [mailing list][mailing list] and start writing code.
```

Comment:
The list does have archives: https://www.freelists.org/archive/concuerror

What I want to achieve is keep the main repository's Issues for bugs and have "everything else" in forks and the mailing list. It will likely never be a problem, but it makes things "cleaner". Does it make sense?

Reply:
First of all, "Issues" is not the same as "Bugs".  Also, sometimes it may not be so clear (to users) what is a bug and what is an issue -- and thus when to report it.  Case in point: #208.

On a related point, I do not see any reason why one should be made to search two archives instead of just one -- for example, why `code:purge` is (still) not supported and what to do about it.

Reply:
Agree. I will revise.